src/main/scala/com/comcast/xfinity/sirius/uberstore/segmented/SegmentedCompactor.scala

```diff
@@ -165,6 +206,7 @@ private [segmented] class SegmentedCompactor(maxDeleteAgeMillis: Long) {
     left.foreach(target.writeEntry)
     right.foreach(target.writeEntry)
     target.setApplied(applied = left.isApplied && right.isApplied)
+    target.setInternallyCompacted(compacted = false)
```

Comment:
Added. I'll be honest, part of the reason I originally let it as 'false' was to avoid writing that test. :)